packages/shared/src/index.ts

```diff
@@ -58,3 +58,40 @@ export const hyphenate = (str: string): string => {
 export const capitalize = (str: string): string => {
   return str.charAt(0).toUpperCase() + str.slice(1)
 }
+
+export const looseEqual = (a: any, b: any): boolean => {
```

Comment:
Let's remove this for now

Reply:
> Would you be interested in adding tests for this module?

Yes.

Reply:
@yyx990803 What would be the optimal strategy for testing `patchEvent`? Should I mock `Element` with `js-dom` and then test by invoking events on the element?